executor/distsql.go

```diff
@@ -910,6 +936,51 @@ func (e *IndexLookUpExecutor) getHandle(row chunk.Row, handleIdx []int,
 	return
 }
 
+type indexLookUpRunTimeStats struct {
+	indexScan    int64
+	tableRowScan int64
+}
+
+func (e *indexLookUpRunTimeStats) String() string {
+	var buf bytes.Buffer
+	indexScan := atomic.LoadInt64(&e.indexScan)
+	tableScan := atomic.LoadInt64(&e.tableRowScan)
+	if indexScan != 0 {
+		buf.WriteString(fmt.Sprintf("index_task_time:%s", time.Duration(indexScan)))
+	}
+	if tableScan != 0 {
+		if buf.Len() > 0 {
+			buf.WriteByte(',')
+		}
+		buf.WriteString(fmt.Sprintf(" table_task_time:%s", time.Duration(tableScan)))
```

Comment:
How about add table_task number and the table worker concurrency information? such as 
```sql
table_task: {num: 100, concurrency: 16, time: 300ms}
```